ThreeDeeFrontend/Pages/Index.razor

```diff
@@ -6,14 +6,18 @@
     <MudText Style="padding-top: 50px;" Typo="Typo.h2">Files</MudText>
     <div style="width: 100%; display: flex;">
         <div style="width: 100%; margin-right: 20px; ">
-            <SearchBar
-                FilteredFilesHaveChanged="async (v) => await OnFilteredValueChanged(v)"/>
+            <SearchBar 
+                Search="@Vm.UpdateFilteredFiles" 
```

Comment:
Was ist das @Vm?

Reply:
Steht für viewmodel. Keine schön Variablenbenennung, aber der Hintergrund ist, dass wenn man mit MVVM Pattern arbeitet quasi jede Komponente nen Viewmodel hat was zudem auch oft aufgerufen/gebunden wird. Deshalb ist es abgekürzt. Ist aber jetzt keine Konvention oder so, man sieht es so und ausformuliert. Im Prinzip isses genau gegen Cleancode Grundsätze das so zu benennen, aber ist halt bei so redundant verwendeten Sachen ne Abwägungsfrage. Aber im Sinne eines Review könnteste jetzt sagen: is nich schön, inkonsistent, gegen interne Standards etc. - mach bitte anders. :-)

Reply:
is nich schön, inkonsistent, gegen interne Standards etc. - mach bitte anders. :-). Hahah spaß ich lass das dann alles mergen^^